tls/s2n_handshake.h

```diff
@@ -104,13 +104,10 @@ struct s2n_handshake_parameters {
     struct s2n_blob client_cert_chain;
     s2n_pkey_type client_cert_pkey_type;
 
-    /* Signature/hash algorithm pairs offered by the client in the signature_algorithms extension */
-    struct s2n_sig_scheme_list client_sig_hash_algs;
+    /* Signature/hash algorithm pairs offered by the peer. */
+    struct s2n_sig_scheme_list peer_sig_scheme_list;
```

Comment:
I love it when a refactor like this just works xD

Is it worth a comment explaining why this is fine? Or would nobody think to separate them again now that we've combined them?

Reply:
I'll add a comment!